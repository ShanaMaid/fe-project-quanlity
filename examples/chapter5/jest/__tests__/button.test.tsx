import * as renderer from 'react-test-renderer';
import * as React from 'react';
import Button from '../Button';

describe('test', () => {
  test('default', () => {
    const component = renderer.create(
      <Button>hello, world</Button>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});