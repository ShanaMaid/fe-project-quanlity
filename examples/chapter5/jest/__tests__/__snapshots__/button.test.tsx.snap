// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test default 1`] = `
<button
  className="button"
>
  hello, world
</button>
`;
